spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'PostsController', type: :request do
+  describe 'GET /posts' do
+    it 'returns http success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('The page displays the posts for given user')
+    end
+  end
+
+  describe 'posts#show' do
+    it 'render the show template' do
+      get '/users/2/posts/2'
+      expect(response).to render_template('show')
+    end
+  end
+end
```

Comment:
- [ ] Kindly implement all of the following tests for each of the actions in each controller:
   - If the response status was correct.
   - If a correct template was rendered.
   - If the response body includes correct placeholder text.
- As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.) requirement.

Reply:
@ShadyShawkat thanks for the review. Kindly note that all the three requirements have been implemented. Please go through again. Thanks.